unleash-yggdrasil/src/state.rs

```diff
@@ -24,10 +24,7 @@ impl EnrichedContext {
             session_id: context.session_id.clone(),
             environment: context.environment.clone(),
             app_name: context.app_name.clone(),
-            current_time: context
```

Comment:
This gets moved to the context resolution code, otherwise this is invoked every single time `is_enabled` is called. Turns out that's rather expensive